src/xebia_email_signature/templates/new_signature.html.jinja

```diff
@@ -0,0 +1,177 @@
ghCsaYzEOGsFwFdRzMWv4aqqPCbFw3Bbq2KFsdHa/O5Mec2Dz6DKsl6LFRPEL95/l7KsZ6n9f/GD+TX9x6o8Ycrq+Lzk4oJgwWsMtZoh7ilHux4jVz1gKjsfX+IAqiWTHCBZ2yNABOIxOp3NrMgnKJCPQEDfB7nb329UTlW0Otg4tSSxetuqZooUCVqHLFAnKMWMMXl55dpisrOatvn4V3ueMw557XCIgCIARHizBtLUwJHOFpRg/OHeCEosxz8H41abWgBWsVnj4RW8DqOf5uHjv2w0MqDsBC1uIU4wS1E39MFtgpcnIP3HKARSwvKCTTx64kKrLYZXFzAdeQdCMIuLFzYemKz8g77IVNYwK1qX2V73y0ivc5pilYKi1MNV3uabQZRK85j4hE5ELG8gIu/1cHQt6uT93yTx3WrOlf0tsy4G5Q1Xe4asYARtfpyhzjIxW7NnRXVjPtqHwYVxhufg++xYv/lP8VR0C3aWDl/dqlAzKqtUdZ80/dvangeduznhEj7eoPk6er1rTXvHLF6Pc+P1ZOmEYoZQURlUq/PEL7GGrVwQZC0dHogK24+zGx3PbYOO6c8UK2hdWjFresGx7YbfqccGMp3S7mEjSLdn/a688x8cf7GyzZczp/aRDEAX9iO3z0QSFQmiCQoB3klsPDpBeq5dNnihcYzLuDsa/+u+RwEnZu8j27Z8R0a4NYZHogyZu4ILUurD7py3VXGZz74z3s1gjAGGR7cXeOp0LB49e8vG9uIdlbvhfwUjmM/T4wnQyvR7ufqlQ+wT6MI1ywcseY9VYUm7LVGuIU41O4WopHE9JC/QroAFydruTh2IALsY6YekG9C36BZNkwvLi4Smngfr6+p9hua4GIfdsk94RxwLJSD/deh37COCT887q3/336eeEYExvniGPh7VcD8XrNoTpJ6jljAGGH6+Q3LWZJZahN+RIWL/ebRE0aHqL26sYKI8uWbrzciBcSDX/OApdMWz1dec5UhKBzfOuLUzCFZI5E9Ypngc6uqwlxhM9z4/JN3PlozUtUpYgFELft5Wo+xooGhjzpiMZ/zPQdpO1uIXkWFbLk1gmDbLZttHa1q5sZedkiOmuD9/VsebjjEBtHDrRxUofu3Pmxkz/3g9Tzxv9u7eKJGUQA4rzXS6ryWH1E5Zcut3TBOwDgf5K4Rc6TByqq5Nqo/jDvnlh9GK+BiOpl1eBszYpkRcGU1qw8BQ7x4HYKEv3NKhprj+k0gDicRo3zsi/KxDdFhX3wG5VvPCf/bOxELLZCwcJvPQlm/2yqv5UVUuFB/4/TBjMuv14u4zK6QsG4vSyK+PJYJvNbXezd8yqrcQhHjVVTwOW8WTpAb+AW7ea4w7gBKOoGEBV45W5zZmBnM8WZkUcuPqPz4ExFX5MwHFhbgFuJxvkI5igB4hQcO/YIu+0Kv4z5BRGUCceHz67PXO+4j4goG/2jfb0pYx84VZ6O4wdXL/YRBbiS1g8QnyrEaZDsirgAomg1k3q1EMTYeSb7lLh2nSCGEdScNhNNIXADHOjzmLXkpqELTwgIsLs5tOc+gKyQLTepQKIs0oatHppoUTLAoRSjCivsQZqG16FQIHsvKrrVD72oti5cLanuo6xXqsrqLhMRTToccscDOviG0ELMkJJbJU1Mq9BVWe9IctcTIJxdVWYwidGEZiz6JkU8sWquTeI5kTWiM2YrjvA9CC1CVOUgjERbSD2LkkwmbK6MqDN28WxEjnzjm2bhfjo1IIpaJGPmkoRZXkYtUWDDy3G80SkIi0IoWF/qKVFjgibn8uNkEFbqbZamLc9dGLiyDstpLQleDLICRaqrSEmEZC21qyW11M5wFqFnvsjURiypDayS31b0Y95taaJmwJLfVxbCHtlaDoGXCAlIldieqzjVtqbCAjDbtLmBv6q3O23JhGaNNtVSJXYNWBXPxSystFxaYPJWfliqxO+BoVTcB3hZhAWkldj6KVN5u2k3aJixpJXY+TtHKeI/aiHEndYymnxS8g2h1YC7vuCRO2yKWiVSJnUe1JdhwcEHbhYUqsZRS2yUF0TmUS2rCyVuZtLUqNHllvvh+q+bbEpoD0Wry9NSP3PZre8QyMYbXULyWDBaWkk6pzV72i42wAEacit+KMUqNulWBi7tSzMAq6KmyPhLX1caSCn7wj89Nfcnr/rGKWAALnctw5nhhtAI9VoEmsTDvdjC/qeS34oS6i3NWM34+EbuIZTJ5Oj/MTwUS2gv7KqNv1yexFRZgM79dzHwb0Wpici4/QgGInXm3gxXRF8r6NTHzrUZNc6TaTgGJdcQCaN6WfRpHoUkUFXvS1NQAgViadzvIzG/s3YBZTFq7omMSgaj4h1xv8J4fOkJY4Ni5YpFbiqqdS9p1PSGJCnSMsACLqyDiigpV4MbS1jBEBTpKWEDEFQVqCkYdi29RSHScsICIK0QqeaphCpmOFBYQcTWJonn+8u5jUY1TBHSssICIKxhIOnMKZ+sTc/nnKSI6WlgA4pJUhB/UFHo0Hvc4/CXwUahLwHCbdFk/KytjOMBVny6p0SfORFP1LT1cF4Hun1JZHxFx2UEqge4ajzhK1RyRugxjIfQSptTRQ5R0Whylag/dpdy/NjestB6jpKLVBG6tCyvh6ZeuFRZIZtWoCqRp1Jgyqp1nQQlg55rcCCm9j7qaeAjKJBHCAsa4rhJx1ai7LC0RL0GZJEZYJrv6cjlEr86vHtU0C2oiboIySZywTDpSYJiGoGyY8vF2mXKvJFZYJhCYJuLUhM5SHKmICbO6TMc1OtUj8cIyqXqwEaX0zW2PYhYxcXQqxj061UOEVQduRWb5m+GqkgZI60GKGghJq4JWdFSnqICbdqnDEWG5UI1kg2yUs4bQlB4MfMcQhqpoNc9Vb5HLOKkUFdNpKrSyq6VViLACgG6jz4j6UwssMEUZ/hIz5eoz3mfhzKd0Zb4vrlZnyz00fwm/1o0CcuL/qSOj7lwl/+cAAAAASUVORK5CYII="
```

Comment:
All the images are inlined here. The service actually has the option to allow for the images to be inlined.
It was an option to allow for a smaller footprint of the signature; Some email clients do not support large signatures.

In addition: it improves the readability and the maintainability of the code.

Reply:
Ok, image src has been changed to urls

Reply:
All images has been changed back to base64